dev/tools/opam-publish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+TAG="$1"
+
+if [ -z "$TAG" ]; then
+  echo "usage: $0 tagname"
+  exit 1
+fi
+
+case $TAG in
+  V*+rc*)
+    OPAM_DIR="--packages-directory=core-dev/packages"
+    OPAM_REPO="--repo=coq/opam-coq-archive"
+  ;;
+
+  *)
+    # defauls are good for the main OPAM repo
+    OPAM_DIR=
+    OPAM_REPO=
+  ;;
+esac
+
+URL="https://github.com/coq/coq/releases/download/$TAG/coq-${TAG##V}.tar.gz"
+
+echo "Publishing tag $TAG for url $URL"
+echo
+echo "The url corresponds to a release asset, not the tarball generated by git,"
+echo "hence you have to download that tarball and re-upload it as an asset!"
```

Comment:
I'm fine using dune-release or target the main archive, but the opam packages then cannot be published as such, since one needs to tweak the dependencies. While this is just automating the existing workflow.

Reply:
Yes, and I didn't mean to say that we should not merge this PR because we may use dune-release in the future.